code/modules/paperwork/clipboard.dm

```diff
@@ -8,119 +11,167 @@
 	w_class = WEIGHT_CLASS_SMALL
 	throw_speed = 3
 	throw_range = 7
-	var/obj/item/pen/haspen //The stored pen.
-	var/obj/item/paper/toppaper //The topmost piece of paper.
 	slot_flags = ITEM_SLOT_BELT
 	resistance_flags = FLAMMABLE
+	/// The stored pen
+	var/obj/item/pen/pen
+	/**
+	 * The topmost piece of paper
+	 *
+	 * Additionaly, all are in contents. This is used for the paper
+	 * displayed on the clipboard's icon and it is the one attacked,
+	 * when attacking the clipboard.
+	 * (As you can't organise contents directly in BYOND)
+	 */
+	var/obj/item/paper/toppaper
 
 /obj/item/clipboard/suicide_act(mob/living/carbon/user)
 	user.visible_message("<span class='suicide'>[user] begins putting [user.p_their()] head into the clip of \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
-	return BRUTELOSS//the clipboard's clip is very strong. industrial duty. can kill a man easily.
+	return BRUTELOSS //The clipboard's clip is very strong. Industrial duty. Can kill a man easily.
 
 /obj/item/clipboard/Initialize()
 	update_appearance()
 	. = ..()
 
 /obj/item/clipboard/Destroy()
-	QDEL_NULL(haspen)
+	QDEL_NULL(pen)
 	QDEL_NULL(toppaper) //let movable/Destroy handle the rest
 	return ..()
 
+/obj/item/clipboard/examine()
+	. = ..()
+	if(pen)
+		. += "<span class='notice'>Ctrl-click to remove [pen].</span>"
+	if(toppaper)
+		. += "<span class='notice'>Alt-click to remove [toppaper].</span>"
```

Comment:
Make removing the pen alt-click and make removing the paper right-click.

Reply:
that'd be inconsistent with pda behaviour